Hazel.UnitTests/Dtls/DtlsConnectionTests.cs

```diff
@@ -315,7 +315,7 @@ public void TestMalformedConnectionData()
 
 
         [TestMethod]
-        public void TestReorderedCertFragmentsConnects()
+        public void TestReorderedHandshakePacketsConnects()
```

Comment:
This is great to see the reordering tolerance! Also happy that that swap method you wrote coming in handy again